src/imap/mod.rs

```diff
@@ -768,17 +768,12 @@ impl Imap {
                 let msg = fetch?;
                 match get_fetch_headers(&msg) {
                     Ok(headers) => {
-                        let (from_id, _, _) = from_field_to_contact_id(
-                            context,
-                            &mimeparser::get_from(&headers),
-                            false,
```

Comment:
Got it, it was about this line! Yes, of course you're right, I missed this when writing this